connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -751,14 +751,8 @@ public Void call() throws Exception {
 
     @Override
     public void connectorConfig(String connName, final Callback<Map<String, String>> callback) {
-        // Subset of connectorInfo, so piggy back on that implementation
         log.trace("Submitting connector config read request {}", connName);
```

Comment:
Seems fine to keep this `trace` log in both cases and keep the implementation only in `AbstractHerder`. Wdyt?

Reply:
The trace logging for other operations (`putConnectorConfig`, `taskConfigs`, etc) is done only in `DistributedHerder`, too. For the consistency sake, I'd leave it as is in this PR.